deploy/addons/headlamp/README.md

```diff
@@ -0,0 +1,41 @@
+## Headlamp Addon
+[Headlamp](https://github.com/kinvolk/headlamp) - Headlamp is an easy-to-use and extensible Kubernetes web UI.
```

Comment:
Have added documentation for the addon.

Reply:
BTW, why not just add the link + the description: [Headlamp](...) is an easy-to-use ....

Reply:
I have also noticed that apparently other addons don't have a README.md file, so not sure if they will accept it for us.